mock-relying-party-ui-esim/configure_start.sh

```
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+#installs the pre-requisites.
+set -e
+
+# Check if $i18n_url_env is not empty
+if [[ -n "$i18n_url_env" ]]; then
+    echo "Downloading pre-requisites install scripts"
+    echo "i18n_url_env is set: $i18n_url_env"
+    wget --no-check-certificate --no-cache --no-cookies $i18n_url_env -O $i18n_path/mock-relying-party-i18n-bundle.zip
```

Shellcheck (0.11.0)

[warning] i18n_path is referenced but not assigned.

(SC2154)